tests/enum_test.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.tests import TestCase
+from superdesk.utils import SuperdeskBaseEnum
+
+
+class EnumToTest(SuperdeskBaseEnum):
+    red = 'red'
+    blue = 'blue'
+    green = 'green'
```

Comment:
Maybe just change the member values to something different than their names - just to make more sure that we are indeed testing the values themselves and not the corresponding member names.